packages/venia-concept/src/components/CreateAccountButton/createAccountButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { shape, string } from 'prop-types';
+import PropTypes from 'prop-types';
+import classify from 'src/classify';
+import Button from 'src/components/Button';
+
+import defaultClasses from './createAccountButton.css';
+
+class CreateAccountButton extends Component {
```

Comment:
Please, right now, create a Button component inside order confirmation page folder, which will receive children property. Add this button to order confirmation page component.

```
class CustomButton extends Component {
    render() {
        const { onClick, children } = this.props;
        return (
            <Button classes={{ root: classes.root }} onClick={onClick}>
                { children }
            </Button>
        );
    }
}
```

Reply:
In future we need to think about adding something like theme property to common Button component. In this case we will be able to write something like this

```
<Button theme="dark">...</Button>
```